planner/core/integration_test.go

```diff
@@ -1489,6 +1489,27 @@ func (s *testIntegrationSuite) TestInvisibleIndex(c *C) {
 	tk.MustExec("admin check index t i_a")
 }
 
+func (s *testIntegrationSuite) TestIssue27648(c *C) {
```

Comment:
Please add some test cases also for SELECT statements. Others LGTM.

Reply:
@time-and-fate , thanks for your review, i will add some select statements test

Reply:
@time-and-fate Hi, I have add more test, can you take time to review again?